xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -207,6 +207,12 @@ These configuration options are not available starting with .NET 5.
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
+ Automatic decompression works only on the top most compression. So if the content is compressed multiple times, only the last compression will be decompressed.
+ For example, for a reponse with `Content-Encoding: gzip, br` header, the content will only be decompressed using Brotli, leaving the still gzipped content in the <xref:System.Net.Http.HttpResponseMessage.Content>.
```

Comment:
> For example, for a reponse with `Content-Encoding: gzip, br` header

"For example, for a response with a `Content-Encoding: gzip, br` header that contains content first compressed with gzip and then that compressed content Brotli compressed,"